packages/seacas/libraries/ioss/src/Ioss_Field.C

```diff
@@ -239,6 +239,47 @@ bool Ioss::Field::transform(void *data)
   return true;
 }
 
+bool Ioss::Field::operator==(Ioss::Field rhs)
+{
```

Comment:
Again, please tell me why these were added and why they output information.

Reply:
Addressed in branch `tkordenbrock-seacas/add-ioshell-compare`.